testing/bta/7.3_03-Fabric/build.gradle

```diff
@@ -8,16 +8,27 @@ base.archivesName = 'UniminedExampleMod'
 version = '1.0-SNAPSHOT'
 
 repositories {
+    ivy {
```

Comment:
This is a temporary solution. I'm hoping to get it published to the Signalum Maven.

Reply:
If this hasn't been removed, you can do it now. I'm not going to maintain it either way. The plan was to move it to BTA or turnip-labs' organization.